tests/core/pyspec/eth2spec/test/altair/block_processing/sync_aggregate/test_process_sync_aggregate.py

```diff
@@ -201,6 +201,74 @@ def test_sync_committee_rewards_duplicate_committee_full_participation(spec, sta
     yield from run_successful_sync_committee_test(spec, state, committee_indices, committee_bits)
 
 
+@with_altair_and_later
+@with_presets([MAINNET], reason="to create duplicate committee")
+@spec_state_test
+def test_sync_committee_rewards_duplicate_committee_only_participate_first_one(spec, state):
+    committee_indices = compute_committee_indices(state)
+    committee_size = len(committee_indices)
+    committee_bits = [False] * committee_size
+
+    # Find duplicate indices that get selected twice
+    dup = {v for v in committee_indices if committee_indices.count(v) == 2}
+    assert len(dup) > 0
+    validator_index = dup.pop()
+    positions = [i for i, v in enumerate(committee_indices) if v == validator_index]
+    committee_bits[positions[0]] = True
+    committee_bits[positions[1]] = False
+
+    # Set validator's balance to 0
+    state.validators[validator_index].effective_balance = 0
+    state.balances[validator_index] = 0
+
+    active_validator_count = len(spec.get_active_validator_indices(state, spec.get_current_epoch(state)))
+
+    # Preconditions of this test case
+    assert active_validator_count < spec.SYNC_COMMITTEE_SIZE
+    assert committee_size > len(set(committee_indices))
```

Comment:
Maybe move this up higher so it's more of a "blocking condition" at the top of the test rather than midway

Reply:
@ralexstokes iiuc the preconditions were about duplicate or nonduplicate?

I changed it to be checked with `is_duplicate_sync_committee` helper in c7e102a5e55ed32525e209d0a01ba5ee7d0ae82a. IMO it's more explicit and more precise than checking with committee size.